nemo/collections/asr/parts/conformer_modules.py

```diff
@@ -69,7 +70,7 @@ def __init__(
         elif self_attention_model == 'abs_pos':
             self.self_attn = MultiHeadAttention(n_head=n_heads, n_feat=d_model, dropout_rate=dropout_att)
         else:
-            raise ValueError(f"Not valid self_attention_model: '{self_attention_model}'!")
+            raise ValueError(f"Not a valid value for 'self_attention_model': {self_attention_model}!")
```

Comment:
Also list out the actual valid values so users dont need to dig into docs

Reply:
done.